src/pages/events/configure-commerce.md

```diff
@@ -142,9 +142,9 @@ Commerce provides two sources for events: observers and plugins. You must specif
 
 1. Optionally create a new OAuth or JWT credential. Then click **Next**.
 
-1. Update the **Event registration name** and **Event registration description** fields. In the **How to receive events** section, under **Option 2**, select the runtime action you created in [Set up App Builder and define a runtime action](./project-setup.md#set-up-app-builder-and-define-a-runtime-action).
+1. Update the **Event registration name** and **Event registration description** fields. Your events are able to be consumed via Journaling API by default. You can also optionally select other consumption methods at this step. Learn more about these options in the [Consuming Events](./consume-events.md) page.
```

Comment:
After changing these lines and image it makes sense to update this page https://developer.adobe.com/commerce/extensibility/events/project-setup/
as **Set up App Builder and define a runtime action** section is not required now and can be removed or marked as optional.



Also, the next lines need to update as this action is not specified anymore:
```
The command initializes a project with a default UI and creates a default Adobe I/O Runtime Action with an internal name of dx-excshell-1/generic. This action will be specified later when configuring your workspace to register events.
```

Reply:
I replaced what was there before and just added a brief section marked as optional in https://github.com/AdobeDocs/commerce-extensibility/pull/219/commits/46fde4431f0d95e4818e3e7ae54aacf21bc3b13a. 

It looks like an optional section replacement was added in https://github.com/AdobeDocs/commerce-extensibility/pull/222 so I can remove my changes in that section, but maybe [project requirements](https://github.com/AdobeDocs/commerce-extensibility/blob/main/src/_includes/project-requirements.md) should be updated in that PR too to mark the parts that are optional

Reply:
Agree, seems the next lines:
```
Be familiar with [Adobe I/O Runtime](https://developer.adobe.com/runtime/docs/guides/getting-started/).

Install the [aio CLI](https://developer.adobe.com/runtime/docs/guides/getting-started/setup/).
```
shouldn't be required as it's possible to configure eventing without it if the customer is using journaling or webhooks.

Reply:
I've removed those 2 lines and the template section in this PR just so those changes will go out with the change to remove the reference to `project-setup.md#set-up-app-builder-and-define-a-runtime-action` in `configure-commerce.md`